docs/maui/Index.md

```diff
@@ -24,17 +24,19 @@ Here is useful information for you getting started with SDK:
 
 ## Supported Platforms
 
-  This SDK supports application built on `.NET7.0`.
+  This SDK supports application built on `.NET8.0`.
 
-  Target Frameworks - `.NET7.0-android` and `.NET7.0-ios`.
+  Target Frameworks - `.NET8.0-android` and `.NET8.0-ios` and `net7.0-windows`.
 
   - **Android** : supports Android API 21 and above.
   - **iOS** : supports 12.0 and above.
+  - **Windows** : supports Windows 11 and Windows 10 version 1809 (build number 10.0.17763) or higher
 
 ## Platform Differences
 
   - **Android** : `CustomTabs` are used to launch browser whenever available, otherwise the system browser is used as a fallback.
   - **iOS** : `SFSafariViewController` is used to launch browser if available, otherwise Safari.
+  - **Windows** : `LocalHostListener` is used listen to the callback from browser.
```

Comment:
mention "The user's default browser is always launched" instead of LocalHostListener. LocalHostListener is internal handling for callback, but we have to mention what type of browser is used here

Reply:
@vasudev-j

Reply:
Done.